tests/test-cases/Group6-VIC-Machine/6-03-Delete.robot

```diff
@@ -43,6 +42,23 @@ Cleanup Delete Tests
     Run Keyword If  '${tempvm}'!='${EMPTY}'  Log  ${output}
     Run Keyword If  '${tempvm}'!='${EMPTY}'  Should Be Equal As Integers  ${rc}  0
 
+Cleanup Delete VCH with non-cVM in VCH folder test
+    Run Keyword And Ignore Error  Cleanup VIC Appliance On Test Server
+
+    Set Environment Variable  TEST_DATACENTER  ${orig-dc}
+
+    # Delete VCH folder and the non-cVM under it
+    ${rc}  ${output}=  Run And Return Rc And Output  govc object.destroy %{VCH-NAME}
+    Log  ${output}
+    Should Be Equal As Integers  ${rc}  0
```

Comment:
this will cause the calls below to leak if it fails

Reply:
Good catch - fixed!

Reply:
https://github.com/vmware/vic/pull/7686